src/components/pages/Navbar/NavbarFeatures/LogoutButton.js

```diff
@@ -1,16 +1,20 @@
 import React from 'react';
 import { useAuth0 } from '@auth0/auth0-react';
 
+import { NavBtnLink } from '../../NavBarLanding/NavBarStyle';
+
 function LogoutButton() {
   const { logout } = useAuth0();
 
+  const logoutAuth = () => {
```

Comment:
It looks like this is still WIP. We need to route 'somewhere'? I'd love to learn more about what the `logout()` fn does.

Reply:
@paulstgermain it sounds like you've worked this out and I know that  @miguelaledesma has worked on it a bit as well. Would love to know how this logout button works.